src/components/shared/SearchInput.js

```diff
@@ -101,31 +107,31 @@ class SearchInput extends Component {
       size
     } = this.props;
 
-    return dom.div(
-      {
-        className: `search-field ${size}`
-      },
-      this.renderSvg(),
-      dom.input({
-        className: classnames({
-          empty: this.shouldShowErrorEmoji()
-        }),
-        onChange,
-        onKeyDown,
-        onKeyUp,
-        onFocus,
-        onBlur,
-        placeholder,
-        value: query,
-        spellCheck: false,
-        ref: "input"
+    const inputProps = {
+      className: classnames({
+        empty: this.shouldShowErrorEmoji()
       }),
-      dom.div({ className: "summary" }, summaryMsg || ""),
-      this.renderNav(),
-      CloseButton({
-        handleClick: handleClose,
-        buttonClass: size
-      })
+      onChange,
+      onKeyDown,
+      onKeyUp,
+      onFocus,
+      onBlur,
+      placeholder,
+      value: query,
+      spellCheck: false,
+      ref: "input"
+    };
+
+    return (
+      <div className="search-field {size || ''}">
```

Comment:
shouldn't it be this: `{`"search-field ${size || ''}"`}`

I'm still new at this game